packages/rocketchat-ui-message/message/popup/messagePopup.coffee

```diff
@@ -108,6 +108,9 @@ Template.messagePopup.onCreated ->
 
 			template.enterValue()
 
+			if template.data.cls is 'search-results-list'
```

Comment:
@rodrigok should we turn this behavior into a config when you initiate messagePopup?